src/EnergyPlus/PluginManager.cc

```diff
@@ -406,8 +406,12 @@ PluginManager::PluginManager(EnergyPlusData &state)
 {
 #if LINK_WITH_PYTHON == 1
     // we'll need the program directory for a few things so get it once here at the top and sanitize it
-    fs::path programPath = FileSystem::getAbsolutePath(FileSystem::getProgramPath());
-    fs::path programDir = FileSystem::getParentDirectoryPath(programPath);
+    fs::path programDir;
+    if (state.dataGlobal->installRootOverride) {
+        programDir = state.dataStrGlobals->exeDirectoryPath;
+    } else {
+        programDir = FileSystem::getParentDirectoryPath(FileSystem::getAbsolutePath(FileSystem::getProgramPath()));
+    }
```

Comment:
Quite simple.  

Before: It would get the current program binary, find the directory, and use that to set Python stuff.  However, this would be problematic when calling via API because the "current program binary" is C:\Python\python.exe or whatever, and we really want to get the E+ install instead.

After: It will now check if the install path has been overridden (by client or automatically by API), and use the install path directly if so.  Otherwise it does the same old thing.